tests/benchdnn/graph/graph.hpp

```diff
@@ -83,7 +85,8 @@ std::string case_to_str(const std::string &json_file,
         const std::map<size_t, std::string> &op_attrs,
         const graph_fpmath_mode_t &fpmath_mode,
         const size_t expected_n_partitions, const int64_t mb,
-        const dnnl_data_type_t dt);
+        const dnnl_data_type_t dt,
```

Comment:
Feels like this `dt` can be incorporated into `dts`. `--dt=f16` should be considered as just a convenient shortcut to `--dt=ID0:f16+ID1:f16+...`. Benchdnn can always report the latter form (new one) instead of the short.

Reply:
Thank you for the suggestion. I feel hesitate to change that as:
- Expanding `--dt=f16` to a list of `ID:DT` pairs requires to deserialize the json file firstly to get all the tensor IDs contained in the file. We may need to adjust the current benchmark flow to achieve that.
- For the repro string, it seems to me that replacing `--dt=f16` with `--dt=ID0:f16+ID1:f16+...` is confusing (and also lengthy) if `--dt=f16` is what the user specified in the command line.